lib/lndclient/LndClient.ts

```diff
@@ -780,7 +780,8 @@ class LndClient extends SwapClient {
       await this.connectPeerAddresses(uris);
     }
 
-    await this.openChannelSync(remoteIdentifier, units, pushUnits);
+    const openResponse = await this.openChannelSync(remoteIdentifier, units, pushUnits);
+    return openResponse.hasFundingTxidStr() ? openResponse.getFundingTxidStr() : openResponse.toObject().fundingTxidBytes.toString();
```

Comment:
Try `return openResponse.getFundingTxidBytes_asB64();` here to return the txId in the correct format. I remember having some issues with gettting grpc bytes fields to give us a string in the desired format but I believe this is what we want here for txids.

Reply:
Thank you! :) This returns txids like `YJpU9BwREyVWLrjB/BEvUTQm7foeKOjXgdhRdMVQKHk=`

Reply:
Sorry I had a brain fart and forgot that txids are in hex not in base64. We already have a helper function for this. You should be able to do `base64ToHex(openResponse.getFundingTxidBytes_asB64())`.